### ISSUE TITLE
When I click on a device I see the process currently running and not the process list

### ISSUE THREAD

**Issue by ninabuffi** [ospin-web-dev/REACTor#384]
All information you provide is _extremely_ helpful to us! Thank you for taking the time.

**Describe the bug**
When I click on a device I see the process currently running and not the process list.

**To Reproduce**
1. I click on the device
2. I land on the current process running and not on the process list. 
This is not 100% reproducible: out of 10 times, 9 times I landed on the running process and 1 time on the process list. 

**Expected behavior**
I should see the process list.

**Screenshots**
No screenshots. 

**Environment**
Production (https://ospin-app.com)

**DeviceID / ProcessID (if applies)**
DeviceID: 5602794b-6eeb-449b-9f6e-868a66fd1547
ProcessID: 360acc26-d983-4e49-9149-b81afbf329e7

**Additional context**

**Comment by DanielSeehausen** [comment]
Hi @ninabuffi 
thank you very much for the bug reports. The other two we are actually going to combine into a feature request as we don't currently have push updates for events not coming from the device (things like sharing a device, etc.)

For this ticket, the application is behaving as intended. When a process is running and a user clicks on the device card, they are taken straight into the running process. When a device has no process running, they are taken to the process list view.

If you think this is non-ideal, we can make a ticket to change it. This was implemented because we figured 90%+ of the time when a user clicks on a running device, their intent is to go into the process.
